src/java.desktop/unix/native/libawt_xawt/xawt/XToolkit.c

```diff
@@ -814,7 +817,7 @@ Window get_xawt_root_shell(JNIEnv *env) {
  */
 
 JNIEXPORT void JNICALL
-Java_sun_awt_motif_XsessionWMcommand(JNIEnv *env, jobject this,
+Java_sun_xawt_motif_XsessionWMcommand(JNIEnv *env, jobject this,
```

Comment:
I'm not sure if this entrypoint is ever accessed externally in a normal JDK.  To be safe, can you make both name changes conditional on STATIC_BUILD?

Reply:
Have made both names conditional.